### DISCUSSION TITLE
[RELEASE] Andes

### DISCUSSION THREAD

**Discussion by vichoca9** [nzp-team/nzportable#772] [Map Releases]
This is my very first map! I *tried* to take inspiration from a real life place (failed doing so jeje), but also for testing what can be done by mapping. Have fun!
![fte-20240608231108-0](https://github.com/nzp-team/nzportable/assets/24655576/a12a7dcb-b817-4a87-bc59-98ee5314ea9e)
![fte-20240608231203-0](https://github.com/nzp-team/nzportable/assets/24655576/30feeb79-3884-4417-89e0-a05691aedf45)
![fte-20240608231227-0](https://github.com/nzp-team/nzportable/assets/24655576/742db458-7c15-4cb7-9e25-14e9783f2ce1)

There are easter eggs, and one buyable ending.
[DOWNLOAD](https://archive.org/download/andes_202406/andes.zip)

**Comment by vichoca9** [comment]
Update: Fixed mapping shenanigans (non integer vertices, zombies not getting upstairs).

**Comment by MotoLegacy** [comment]
Tagging as Meets Install Standard!

**Comment by Phynix0027** [comment]
Can't find the easter eggs 🫠🫠🫠

**Reply by ghost** [reply]
I've done some investigating the Easter egg is there but is very broken,you can only activate it on round 1 with the use of noclip, won't activate on any other round. And you have to restart the map as soon as you get into it

**Comment by nonMat** [comment]
<img width="1920" height="1080" alt="immagine" src="https://github.com/user-attachments/assets/683c2126-9b7f-4fe4-883a-b887e78253af" />
This was very fun!
Awesome setting!
Couldn't figure out the easter egg!
